bundles/com.zeligsoft.domain.ngc.ccm.generator/src/com/zeligsoft/domain/ngc/ccm/generator/xtend/mainTransformAtcd.ext

```diff
@@ -553,7 +566,7 @@ idl::Definition visit(Definition self, emf::EObject container) :
 idl::Definition visit(Definition self, CXModule container) :
 	let idlModule = new idl::Module :
 		idlModule.setName(container.name) ->
-		idlModule.definitions.add(self) ->
+		idlModule.definitions.add(self.copy()) ->
```

Comment:
I'm having `copy` confusion. Line 234 renamed `copy` to `copyEOb`, but here, we're introducing a call to `copy`. Should this be `copyEObj`?